shotover-proxy/src/transforms/protect/key_management.rs

```diff
@@ -118,5 +117,5 @@ async fn private_cached_fetch(
 pub struct KeyMaterial {
     pub ciphertext_blob: Bytes,
     pub key_id: String,
-    pub plaintext: Key,
+    pub plaintext: Vec<u8>,
```

Comment:
We can just directly use Key now:
![image](https://user-images.githubusercontent.com/5120858/174505063-ce013f27-4618-4096-afba-d353fecfc6c4.png)